scenarios/p4rt-ifaces-config.xml

```diff
@@ -0,0 +1,18 @@
+<!--
+ ~ SPDX-License-Identifier: Apache-2.0
+ ~ SPDX-FileCopyrightText: 2022-present Open Networking Foundation <info@opennetworking.org>
+  -->
+<scenario name="p4rt-ifaces-config" description="Configure UE pool on the UP4 interfaces table">
+    <group name="P4rt-Ifaces-Config">
+        <step name="P4rt-Ifaces-Config-Push-Uepool"
+              exec="${DOCKER_COMPOSE_CMD} exec -T p4rt p4rt-up4-flows program --add-iface-uepool --ue-pool 17.0.0.0/16 --ue-count 0 --election-id 105"/>
+        <step name="P4rt-Ifaces-Config-Check-Uepool" requires="^"
+              exec="onos-cli-grep ${OCI} up4:read-interfaces 'UpfInterface(type=CORE, prefix=17.0.0.0/16'"/>
```

Comment:
I remember in some files /24. Do we need to update elsewhere ?

Reply:
No that's ok. The `/24` you see in pfcp/p4rt scenarios is to be able to generate non-overlapping UE IPs when we have multiple eNBs (i.e., for eNB 1 we generate UE IP address into the 17.0.0.0/24 subnet, while for eNB2 17.0.1.0/24).